tests/storage_pool/storage_pool_test.go

```diff
@@ -26,33 +26,71 @@ var _ = BeforeSuite(func() {
 	InitInstance()
 })
 
-var _ = Describe("{StoragePoolResize}", func() {
+var _ = Describe("{StoragePoolExpandDiskResize}", func() {
+
+	Context("When the author has both names", func() {
```

Comment:
what is the author?

Reply:
ah, sorry, I've copied it from the example and forgotten to remove